extract_recipes.py

```diff
@@ -0,0 +1,220 @@
+# coding: utf-8
+
+import bs4
+import json
+import re
+import requests
+import os
+import time
+
+from collections import namedtuple
+from fractions import Fraction
+from functional import seq
+from string import capwords
+from unicodedata import numeric
+
+## Sample data for validation
+
+raw_episode = requests.get('https://www.bingingwithbabish.com/recipes/parksandrecburger')
+soup = bs4.BeautifulSoup(raw_episode.content, 'html.parser')
+recipe_locations = soup.find_all(['h1','h2','h3','h4','h5'], string='Ingredients')
+loc1 = recipe_locations[0]
+method = loc1.find_next(['h1','h2','h3','h4','h5']).string
```

Comment:
This section is for showing a sample of the data in the notebook, but it really wouldn't be useful anymore in this context.